### PR TITLE
Created Contact me section of Portfolio mobile version

### PR DESCRIPTION
**Pull request by megha-n-bodke**
- Created contact me section using flexbox.
- Applied CSS effects

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,272 +9,352 @@
...
+
+      <!-- contact us section starts here -->
+      <footer>
+        <section class="contactus-container">
+          <h1 class="contact-heading">Contact me</h1>
+          <p class="contact_description">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+          <form class="input-data" action="https://formspree.io/f/xqknvnzp" method="post">
+            <input type="text" name="username" id="username" maxlength="30" value="Yeremias" required>
+            <input type="email" name="email" id="email" value="natajayanj14@gmail.com">
+            <textarea id="message" placeholder="Write your message here"></textarea>
```
Comment (omarsalem7) [inline index.html]: - [x] Kindly add the `required` attribute to these input fields have the required attribute to activate the validation on submit.

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @megha-n-bodke 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- Descriptive pull request ✔️
- No linter errors ✔️
- Your design stick to the Figma template. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (megha-n-bodke) [reply index.html]: updated review changes

**Review by youmari — APPROVED** [review]
# Project Approved :trophy: :tada: 🟢 

Hi @megha-n-bodke 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Details PR title and description.
:heavy_check_mark: Good commit messages.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**